src/app/[locale]/(pages)/ripristino-accesso/inserisci-codice/page.tsx

```diff
@@ -20,6 +21,8 @@ const ReactivateCode = (): React.ReactElement => {
 
```

Comment:
When I was testing I saw that the error remains even when the user changes code, don't you think it should be changed?

Reply:
You're right, fixxed.